gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,655 @@
+{
+    "author_info": {
```

Comment:
We'll need scope variables if people are monitoring multiple gitlab instances. Maybe `gitlab_host` ?